designer/client/components/Visualisation/Visualisation.tsx

```diff
@@ -39,6 +39,7 @@ export function Visualisation(props: Props) {
   const wrapperStyle = layout && {
     width: layout?.width,
     height: layout?.height,
+    top: "10px",
```

Comment:
is there nothing from gov uk styles that you can use to improve their styles. We don't tend to break outside of these styles very often. Its better to follow their standards. https://design-system.service.gov.uk/styles/layout/

Reply:
so i can remove this top line, the problem is non of there styles are fixed which is why i had to edit some of them a bit